### ISSUE TITLE
Bug report + fix: forward_kwargs not used in all self.model calls in integrated_gradients

### ISSUE THREAD

**Issue by sanjass** [SeldonIO/alibi#524]
In lines [808](https://github.com/SeldonIO/alibi/blob/master/alibi/explainers/integrated_gradients.py#L808), [856](https://github.com/SeldonIO/alibi/blob/master/alibi/explainers/integrated_gradients.py#L856) and [926](https://github.com/SeldonIO/alibi/blob/master/alibi/explainers/integrated_gradients.py#L926) in `integrated_gradients.py` the `forward_kwargs` argument is missing from the `self.model` function calls. 

The following change resolved the issue for me: `self.model(X)` -> `self.model(X, **forward_kwargs)`. Note in some cases `X` is `inputs`.

**Comment by jklaise** [comment]
Good catch and thanks for the PR!

Although now I wonder how our (transformers example)[https://docs.seldon.io/projects/alibi/en/latest/examples/integrated_gradients_transformers.html] worked with this bug... @gipster

**Comment by sanjass** [comment]
In the first example of [transformers example](https://docs.seldon.io/projects/alibi/en/latest/examples/integrated_gradients_transformers.html)
 the `self.model` call works even if the `attention_mask` is `None`, so it wasn't throwing any errors despite `forward_kwargs` not being used. You can confirm this by printing out the `attention_mask` in the model `call`; sometimes it's `None`, sometimes it's what it's supposed to be (bc `forward_kwargs` is used only in some calls). 

In my case, my model had `Input` layers so a `None` input for `attention_mask` (and `input_type_ids` for that matter) wouldn't work and I was getting errors such as `TypeError: Inputs to a layer should be tensors. Got: None`. With the above change it worked as expected in my case.

**Comment by sanjass** [comment]
@jklaise thanks for assisting with this issue! I see the PR has been merged to master, but is there an estimate when would the fix be released, so it works after `pip install alibi`? I see the "licences" CI check fails, is there anything I can do about that?

**Comment by jklaise** [comment]
@sanjass we can release a patch release today or tomorrow, in the meantime you can install from github directly via ```pip install git+https://github.com/SeldonIO/alibi.git 
```

The licences is an easy fix which we'll do prior releasing, no need to worry about that :)

**Comment by jklaise** [comment]
Fixed by #525.
